mava/configs/logger/base_logger.yaml

```diff
@@ -0,0 +1,14 @@
+# --- Logging options ---
+
+use_tf: False # Whether to use tensorboard logging.
+use_sacred: True # Whether to use sacred logging.
+base_exp_path: results # Base path for sacred logging.
+system_name: ~
```

Comment:
Since we will always have systems we can just set the system name in the system logger files.